src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-29">
+      <div class="calendar__day calendar__day--1 calendar__day--sun"></div>
+      <div class="calendar__day calendar__day--2"></div>
+      <div class="calendar__day calendar__day--3"></div>
+      <div class="calendar__day calendar__day--4"></div>
+      <div class="calendar__day calendar__day--5"></div>
+      <div class="calendar__day calendar__day--6"></div>
+      <div class="calendar__day calendar__day--7"></div>
+      <div class="calendar__day calendar__day--8"></div>
+      <div class="calendar__day calendar__day--9"></div>
+      <div class="calendar__day calendar__day--10"></div>
+      <div class="calendar__day calendar__day--11"></div>
```

Comment:
Each day has no modifiers, only class (eg. calendar__day). It means you sholud remove modifier like `calendar__day--11`

 
![image](https://github.com/mate-academy/layout_calendar/assets/91946036/8fc37d38-ddbf-419a-a66b-004b79d255b2)